qiime2/sdk/result.py

```diff
@@ -275,11 +275,12 @@ def import_data(cls, type, view, view_type=None):
 
         provenance_capture = archive.ImportProvenanceCapture(format_, md5sums)
         return cls._from_view(type_, view, view_type, provenance_capture,
-                              validate_level='max')
+                              level='max')
```

Comment:
I would revert this particular parameter back to `validate_level` (it's not public, and clarifies things at this call-site).